eth/gasprice/optimism-gasprice.go

```diff
@@ -0,0 +1,116 @@
+package gasprice
+
+import (
+	"context"
+	"math/big"
+	"sort"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/ethereum/go-ethereum/params"
+	"github.com/ethereum/go-ethereum/rpc"
+)
+
+var (
+	// TODO: What's the recommended way to make this a configurable parameter?
+	MinSuggestedOptimismPriorityFee = big.NewInt(1e8 * params.Wei) // 0.1 gwei
```

Comment:
This probably should be the same value as used for the `--miner.gasprice` flag since that's the minimum price required for a transaction to be included at all.  Alternatively it could be a new CLI flag that adds its value into `gasprice.Config` which gets passed through to `NewOracle`.

Reply:
As this is a fee "suggestion" I'd rather not override miner.gasprice, which is a hard minimum under which the tx would just be discarded.  Let me look into gasprice.Config.

Reply:
Why would you want to always pay more than the minimum that would be accepted though? Making it separate probably makes sense so there's more control but I think I'd be setting it to be the same as the minimum miner price to minimise fees paid.

Reply:
@ajsutton the value is often described as a "tip", and the way I see it is a tip shouldn't be mandatory, but it's a reasonable question to ask as it's certainly the perogative of the block builder to reject a tx based on arbitrary logic. Until there is more than one builder though, maybe it's best to be permissive?

P.S.: it is in fact true that one can already specify a priority fee much lower than any existing suggestions (be they in wallets or as implemented in Ethereum) and the tx will still go through, since geth has such a low IgnorePrice by default. I do this all the time!

Reply:
Yeah my thinking is that for L2s where there's a big focus on keeping fees low (and the baseFee isn't burnt) then you'd want the estimation to always give the minimum tip that can be paid to be included which with a centralised sequencer would be the same as its `miner.gasprice`.

But it does make sense to keep them as separate config items because of the extra flexibility it gives. You can always set both options to the same value if you want to recommend paying the minimum tip that you'd accept.